unitex/src/fr/umlv/unitex/frames/ConjugaisonFactory.java

```diff
@@ -0,0 +1,12 @@
+package fr.umlv.unitex.frames;
+
+import leximir.Shell;
+
+public class ConjugaisonFactory {
```

Comment:
What about to use "Conjuga**tion**" instead of "Conjuga**ison**"?

Reply:
All of conjugaison's file is not mine. I integrate it with my project for test but i think there are no release for now. So I need no delete it?